src/Sylius/Bundle/CoreBundle/Export/Reader/ORM/UserReader.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Export\Reader\ORM;
+
+use Doctrine\ORM\AbstractQuery;
+use Doctrine\ORM\EntityRepository;
+use Psr\Log\LoggerInterface;
+use Sylius\Bundle\CoreBundle\Export\Reader\ORM\Processor\UserProcessorInterface;
+use Sylius\Component\ImportExport\Model\JobInterface;
+use Sylius\Component\ImportExport\Reader\ReaderInterface;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class UserReader implements ReaderInterface
+{
+    /**
+     * @var EntityRepository
+     */
+    private $userRepository;
+
+    /**
+     * @var UserProcessorInterface
+     */
+    private $userProcessor;
+
+    /**
+     * @var array
+     */
+    private $metadata = array();
+
+    /**
+     * @param UserProcessorInterface $userProcessor
+     * @param EntityRepository       $userRepository
+     */
+    public function __construct(
+        UserProcessorInterface $userProcessor,
+        EntityRepository $userRepository
+    ) {
+        $this->userProcessor = $userProcessor;
+        $this->userRepository = $userRepository;
+        $this->metadata['result_code'] = 0;
+        $this->metadata['offset'] = 0;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getType()
```

Comment:
The order of methods should be the same as in the interface (will be easier to follow similar classes).